src/dungeon_data.c

```diff
@@ -59,7 +59,7 @@ struct Dungeon *get_dungeon_f(PlayerNumber plyr_num,const char *func_name)
 {
     if ((plyr_num < 0) || (plyr_num >= DUNGEONS_COUNT))
     {
-        ERRORLOG("%s: Tried to get non-existing dungeon %ld!",func_name,(long)plyr_num);
+        ERRORLOG("%s: Tried to get non-existing dungeon %ld!", func_name, (long)plyr_num);
```

Comment:
if you remove the typecast here also replace the %ld by %d

Reply:
Done